controllers/apps/operations/rebuild_instance.go

```diff
@@ -774,7 +774,7 @@ func (r rebuildInstanceOpsHandler) instanceIsAvailable(
 	if !targetPod.DeletionTimestamp.IsZero() {
 		return false, nil
 	}
-	isFailed, isTimeout, _ := intctrlutil.IsPodFailedAndTimedOut(targetPod)
+	isFailed, isTimeout, _ := intctrlutil.IsPodFailedAndTimedOut(targetPod, true)
```

Comment:
Why modify the check inside instanceIsAvailable?

if Pod is not scheduled,  instanceIsAvailable should not return true.

Reply:
When rebuilding an instance, some scenarios require waiting for the pod to become available to perform data recovery operations. For example, OB. However, there is a high probability of unsuccessful scheduling for a period of time, including pull image failed. Maybe wait for a period of time or solve the scheduling failure problem to continue. So I think we can ignore the detection of pod scheduling failures when rebuilding instances.